### ISSUE TITLE
Gnome Shell 3.20.1 white bar on left of background

### ISSUE THREAD

**Issue by smiffy6969** [snwh/paper-gtk-theme#318]
Hiya, First off, best theme for gnome shell by far.... love your work!

I have just installed ubuntu gnome 16.04 and update gnome-shell from stable to 3.20.1.....

The only real issue I am seeing is a thin, say 2px wide white bar down left of the background on desktop. It isn't there when dash is live, or when a window is maximized, only on background (even the notification drawer bottom left seems to skate over the top of it. as well as top panel).

Not a deal breaker just a bit annoying really. I assume it is a padding or margin or something similar.

Any chance you can point me in the right direction, If you can get me close to wear it may be in the style files I should be able to find it.

thanks

Paul

**Comment by maxkoryukov** [comment]
@smiffy6969 JAKE)))

**Comment by smiffy6969** [comment]
Jake is my hero!

**Comment by mspoehr** [comment]
Can confirm.  Have the same problem running gnome 3.20.1.  When I disable desktop icons the line disappears... but that is only a band-aid.

**Comment by smiffy6969** [comment]
Yes... disabling icons on desktop gets rid of it. Awesome.... for now.

**Comment by arcadian09** [comment]
I can confirm that I have the same issue on Fedora 24, gnome 3.20.4.

issue number:
https://github.com/snwh/paper-gtk-theme/issues/415

**Comment by YappyYellowYak** [comment]
```
Remove .paned,
paned {
  border: solid #c4c4c4;
  border-left-width: 1px; }
```
This seems to be the problem, removing it from gtk.css/gtk-dark.css should work

**Comment by otarassov** [comment]
Removed the line above from `/usr/share/themes/Paper/gtk-3.20/gtk.css` and when selecting paper gtk still shows on top thin white line.

Do i remove it from both gtk.css abd gt-dark.css even though I am not on dark theme?

**Comment by YappyYellowYak** [comment]
`
searchbar,
.search-bar, .location-bar,
.inline-toolbar {
  background-color: #f2f2f2;
...
  border: 1px solid #c4c4c4;
...
}
`
The top line seems to be here, but I can't nail the problem down to a specific line. Deleting all of them seems to work.

**Comment by PhoeniZ** [comment]
I have the same issue in top bar, but the solution of background-color_ transparent doesent work :(
